csi/src/meta_data.rs

```diff
@@ -1007,21 +1082,22 @@ impl MetaData {
             BindMountMode::Multiple => {
                 let update_res = self
                     .etcd_client
-                    .update_existing_kv(&volume_mount_path_key, &mount_path_value_in_etcd);
+                    .update_existing_kv(&volume_mount_path_key, &mount_path_value_in_etcd)
+                    .await;
                 match update_res {
                     Ok(pre_mount_paths) => {
                         debug_assert!(
                             !pre_mount_paths.contains(target_path),
                             "the previous mount paths={} of volume ID={} \
-                                should not contain new mount path={}",
+                            should not contain new mount path={}",
```

Comment:
extra 4 spaces indentation before the second text line

Reply:
Done